docker/build-tools/Dockerfile.ubi9

```diff
@@ -0,0 +1,923 @@
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is used to build Istio's primary build container, which contains all the tools
+# necessary to perform and all build activities in all Istio repos.
+# The container currently supports linux/amd64 and linux/arm64 builds.
+#
+# The container is built using different contexts, one per execution environment (plain binaries, Ruby, nodejs), and
+# then combined into the final image.
+#
+# We pin versions of stuff we install. Modify the various XXX_VERSION variables within the individual build contexts
+# in order to control these versions.
+
+################
+# Binary tools
+################
+ARG BASE_IMAGE=registry.access.redhat.com/ubi9/ubi:9.2-696
```

Comment:
I don't think we should do this. It doesn't benefit the project and adds high maintenance cost

Reply:
I see. yes, it adds high maintenance cost. I can mark this PR as a draft and and answer your question about any value later. Currently, this is an experimental setup and I'd like to collect some feedback of this alternative base.